VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -1,139 +1,95 @@
-import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import { ReactTestInstance } from 'react-test-renderer'
-
-import { context, render, waitFor } from 'testUtils'
-import { PickerItem } from './VAModalPicker'
+import { context, render, screen, fireEvent } from 'testUtils'
 import VAModalPicker from './VAModalPicker'
-import TextView from 'components/TextView'
-import BaseListItem from 'components/BaseListItem'
 import { InitialState } from 'store/slices'
-import { RenderAPI } from '@testing-library/react-native'
 
 context('VAModalPicker', () => {
-  let component: RenderAPI
-  let testInstance: ReactTestInstance
   let selected: string
-  let pickerOptions: Array<PickerItem>
-  let doneButton: any
-  let cancelButton: any
-  let selectionButtons: any
-
-  const initializeTestInstance = async (
-    selectedValue: string,
-    labelKey?: string,
-    helperTextKey = '',
-    error = '',
-    isRequiredField = false,
-    testID = '',
-    isRunning = false,
-  ): Promise<void> => {
-    selected = selectedValue
-    const setSelected = (updatedSelected: string) => {
-      selected = updatedSelected
-    }
 
+  const initializeTestInstance = (
+    labelKey: string = '',
+    helperTextKey: string = '',
+    isRequiredField: boolean = false,
+    error: string = ''
+  ) => {
     const props = {
-      selectedValue: selectedValue,
-      onSelectionChange: setSelected,
-      pickerOptions,
+      selectedValue: 'js',
+      onSelectionChange: (updatedSelected: string) => {
+        selected = updatedSelected
+      },
+      pickerOptions: [
+        { label: 'Java', value: 'java' },
+        { label: 'JavaScript', value: 'js' },
+        { label: 'JavaScript2', value: 'js2' },
+        { label: 'JavaScript3', value: 'js3' },
+      ],
       labelKey,
       helperTextKey,
       error,
       isRequiredField,
-      testID,
+      testID: ''
     }
 
-    component = render(<VAModalPicker {...props} />, {
+    render(<VAModalPicker {...props} />, {
       preloadedState: {
         accessibility: {
           ...InitialState.accessibility,
-          isVoiceOverTalkBackRunning: isRunning,
+          isVoiceOverTalkBackRunning: false,
```

Comment:
you can remove the state stuff all together here as none of that is likely needed

should be render(<VAModal Picker {...props} />)

Reply:
Updated.